agent/api/task/task.go

```diff
@@ -96,11 +96,18 @@ const (
 	ipcModeSharable = "shareable"
 	ipcModeNone     = "none"
 
-	// firelensConfigBindFormatFluentd and firelensConfigBindFormatFluentbit specifies the format of the firelens
+	// firelensConfigBindFormatFluentd and firelensConfigBindFormatFluentbit specify the format of the firelens
 	// config file bind mount for fluentd and fluentbit firelens container respectively.
 	// First placeholder is host data dir, second placeholder is taskID.
 	firelensConfigBindFormatFluentd   = "%s/data/firelens/%s/config/fluent.conf:/fluentd/etc/fluent.conf"
 	firelensConfigBindFormatFluentbit = "%s/data/firelens/%s/config/fluent.conf:/fluent-bit/etc/fluent-bit.conf"
+
+	// firelensS3ConfigBindFormatFluentd and firelensS3ConfigBindFormatFluentbit specify the format of the bind mount
+	// for the firelens config file downloaded from S3. First placeholder is host data dir, second placeholder is taskID,
+	// third placeholder is the s3 config path inside the firelens container.
+	firelensS3ConfigBindFormatFluentd   = "%s/data/firelens/%s/config/s3.conf:%s"
+	firelensS3ConfigBindFormatFluentbit = "%s/data/firelens/%s/config/s3.conf:%s"
```

Comment:
How about `external.conf` or `include.conf` or something besides `s3.conf`. Since we talked about adding support for pulling from SSM in the future.

Reply:
sure. changed to external.conf